service-pdf/composer.json

```diff
@@ -14,6 +14,7 @@
         "php": "^8.0",
         "aws/aws-sdk-php": "^3.204.5",
         "mikehaertl/php-pdftk": "0.2.1",
+        "mikehaertl/php-shellcommand": "1.6.4 as 1.0.2",
```

Comment:
did it need to be redirected? or can this be a direct replacement?

Reply:
I believe you have to explicitly override it. This command tells it to use 1.6.4 instead of 1.0.2.  If you just required 1.6.4 here, my (possibliy limited) understanding is that it could still decide to use 1.0.2